doc/manuals/admin/build_source.md

```diff
@@ -42,6 +42,16 @@ commands that require root privilege):
         tar xfvz v1.0.2.tar.gz
         sudo mv rapidjson-1.0.2/include/rapidjson/ /usr/local/include
 
+* Install microhttpd from sources:
```

Comment:
Maybe we can add some comment about the configure options, e.g:

"Install microhttpd from sources (the `./configure` command below shows the recommened build configuration to get minimum footprint, but if you are and advanced user, you can configure as you prefer)"